OpenSim/Examples/ExampleHopperDevice/buildTestbedModel.cpp

```diff
@@ -1,7 +1,5 @@
-#ifndef _helperMethods_h_
-#define _helperMethods_h_
 /* -------------------------------------------------------------------------- *
- *                         OpenSim:  helperMethods.h                          *
+ *                   OpenSim:  buildTestbedModel.cpp                          *
```

Comment:
The file used to contain several other methods and had a different flavor. It makes sense to change the name given the current contents of the file.